converters/toZigbee.js

```diff
@@ -883,6 +883,60 @@ const converters = {
         },
     },
 
+    livolo_switch_on_off: {
+        key: ['state', 'state1', 'state2'],
```

Comment:
We need to get rid of `state1` and `state2`, otherwise this device cannot be integrated with Home Assisstant.

I propose to add an extra parameter to `convert` containing the postfix (`left`/`right`) => `convert: (key, value, message, type, postfix) => {`. Based on that we can than switch the channel. Please let me know what you think about this, then I can make this change.

Reply:
I don't use Home Assistant and don't know how they will send commands to zigbee2mqtt. How will it send postfix?

Reply:
My device:
https://www.livolo.com/en/catalog/product/view/id/90/s/c702z-z/category/7/

Reply:
The postfix will be parsed from the topic. E.g. `zigbee2mqtt/[DEVICE_ID]/left/set`  `zigbee2mqtt/[DEVICE_ID]/right/set`

Reply:
Looks good. But how Home Assistant does currently work with 2 button switches?

Reply:
Ok. Please, make the necessary change.